localstack-core/localstack/services/ses/provider.py

```diff
@@ -524,6 +527,42 @@ def clone_receipt_rule_set(
 
         return CloneReceiptRuleSetResponse()
 
+    @handler("SetIdentityHeadersInNotificationsEnabled")
+    def set_identity_headers_in_notifications_enabled(
+        self,
+        context: RequestContext,
+        identity: Identity,
+        notification_type: str,
+        enabled: bool,
```

Comment:
We have a [failing unit](https://github.com/localstack/localstack/actions/runs/16622232624/job/47040543356?pr=12823#step:9:22431) test because the arguments for this function were modified from superclass.

```suggestion
        notification_type: NotificationType,
        enabled: Enabled,
```

I will fix it and re-run the pipeline